network/p2p/tracer/gossipSubMeshTracer_test.go

```diff
@@ -0,0 +1,179 @@
+package tracer_test
```

Comment:
Should we add another test case that ensures we log a warning whenever a node is not in the identity state?

Reply:
https://github.com/onflow/flow-go/pull/3970/commits/0097d6752cd80e054ff3c917b89d77efc9e9e1c1